### PR TITLE
Index page updates. Update to metadata.

### PR DESCRIPTION
**Pull request by DougManuel**
- Update to index page formatting
    - logo added.
    - PDF and EPUB download buttons
    - Links to GitHub
- Add metadata for versioning
    - versioning for documentation in addition to dictionary
- address: https://github.com/Big-Life-Lab/PHES-ODM-Doc/issues/42
- [bug] fix URL links.

### CODE REVIEW

File: qmd/_quarto.yml [hunk 1]
```diff
@@ -4,26 +4,38 @@ project:
 
 book:
   title: "Public Health Environmental Surveillance Open Data Model (PHES-ODM) Documentation"
-  subtitle: "Version 2.0.0"
+  output-file: "ODM-documentation-v2.0.1"
+  version: "2.0.1"
+  subtitle: |
+      Dictionary v2.0.0. <br>Documentation V2.0.1.
```
Comment (rvyuha) [inline qmd/_quarto.yml:10]: image
Using the br between versions looks strange to me. I believe keeping them on the same line would be better.

File: qmd/_dictionary-version.qmd [hunk 2]
```diff
@@ -6,4 +6,4 @@ dictionary_version_pattern <- "ODM_dictionary_(.*?).xlsx"
 
 dictionary_version_used <- regmatches(dictionary_full_file_name, regexec(dictionary_version_pattern, dictionary_full_file_name))[[1]][2]
 ```
-ODM version `r dictionary_version_used`.
\ No newline at end of file
+ODM `r dictionary_version_used`.
```
Comment (rvyuha) [inline qmd/_dictionary-version.qmd:9]: In the quarto.yml you use Dictionary to specify the dictionary version. I think ODM should be changed to Dictionary nary to be standard across pages.

File: qmd/_quarto.yml [hunk 3]
```diff
@@ -4,26 +4,38 @@ project:
...
         - parts.qmd
         - sets.qmd
-        #- languages.qmd
-        #- translations.qmd
+          #- languages.qmd
+          #- translations.qmd
         - tables.qmd
-        #- partTypes.qmd
-
+          #- partTypes.qmd
```
Comment (rvyuha) [inline qmd/_quarto.yml:34]: Do we want to keep these commented out or should we remove them now?

File: qmd/_quarto.yml [hunk 4]
```diff
@@ -4,26 +4,38 @@ project:
...
+  page-footer:
+    left: |
+      #Dictionary v2.0.0<br>Documentation V2.0.1
```
Comment (rvyuha) [inline qmd/_quarto.yml:37]: What is the purpose of "#"? It is displayed on the html pages and appears to serve no purpose.

File: qmd/_dictionary-version.qmd (hunk 2 above)
Comment (DougManuel) [reply qmd/_dictionary-version.qmd:9]: Agree. Can you make that change?

File: qmd/_quarto.yml (hunk 3 above)
Comment (DougManuel) [reply qmd/_quarto.yml:34]: Let's remove them.

File: qmd/_quarto.yml (hunk 4 above)
Comment (DougManuel) [reply qmd/_quarto.yml:37]: I was going to comment out. What do you think? Should we have as a footer? Or just at the index page and the pages with the specific lists.

Comment (rvyuha) [reply qmd/_quarto.yml:37]: I think just having it on top of pages that use it is sufficient? @yulric what do you think?

Comment (yulric) [reply qmd/_quarto.yml:34]: Most website put it in the footer I think. The footer is where the website metadata lives, extra information not directly related to the content.

I would put it in the footer.

**Review by yulric — APPROVED** [review]

**Pull request merged**